File: C++/UserInput.cpp
#include <iostream>
using namespace std;

int main() {
    // Declare a variable to store user input
    string userInput;
    
    // Prompt the user for input
    cout << "Enter something: ";
    
    // Read a line of text from the user
    getline(cin, userInput);
    
    // Display the user's input
    cout << "You entered: " << userInput << endl;
    
    return 0;
}

File: java/UserInput.java
import java.util.Scanner;

public class UserInput {
    public static void main(String[] args) {
        // Create a Scanner object to read input
        Scanner scanner = new Scanner(System.in);
        
        // Prompt the user for input
        System.out.print("Enter something: ");
        
        // Read a line of text from the user
        String userInput = scanner.nextLine();
        
        // Display the user's input
        System.out.println("You entered: " + (userInput + 75));
        
        // Close the Scanner
        scanner.close();
    }
}
